tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -66,6 +66,29 @@ def _run(self):
                             GlusterBrickDir()
         gluster_brick_dir.save()
 
+        try:
+            NS.tendrl.objects.ClusterNetworkDetails(
+                integration_id=NS.tendrl_context.integration_id
+            ).load()
+        except etcd.EtcdKeyNotFound:
+            try:
+                node_networks = NS._int.client.read(
+                    "nodes/%s/Networks" % NS.node_context.node_id
+                )
+                # TODO: (team) this logic needs to change later
+                # multiple networks supported for gluster use case
+                node_network = NS.tendrl.objects.NodeNetwork(
+                    interface=node_networks.leaves[0].value
+                ).load()
+                NS.tendrl.objects.ClusterNetworkDetails(
+                    integration_id=NS.tendrl_context.integration_id,
+                    public_network=public_network,
```

Comment:
var `public_network` and `cluster_network` is not defined anywhere and yet it is used here